### PR TITLE
PREAPPS-4538 : Allow authenticated user to switch between Modern and Classic UI

### PR DESCRIPTION
**Pull request by auminpatel**

### CODE REVIEW

File: WebRoot/js/zimbraMail/core/ZmZimbraMail.js [hunk 1]
```diff
@@ -2333,6 +2333,9 @@ function(parent, parentElement, adminUrl) {
 	    mi.addSelectionListener(new AjxListener(null, ZmZimbraMail.adminLinkCallback, adminUrl));
 	}
 
+	mi = menu.createMenuItem("standardHtmlLink", {text: ZmMsg.modernClient});
```
Comment (madhavdhuppe) [inline WebRoot/js/zimbraMail/core/ZmZimbraMail.js]: can we change "standardHtmlLink" to modernLink

Comment (auminpatel) [reply WebRoot/js/zimbraMail/core/ZmZimbraMail.js]: Done

File: WebRoot/js/zimbraMail/core/ZmZimbraMail.js [hunk 2]
```diff
@@ -2851,16 +2854,18 @@ function(url) {
...
 	var urlParams = {
 		path: appContextPath,
 		qsArgs: {
-			client: "standard"
+			client: "modern"
 		}
 	};
 	var url = AjxUtil.formatUrl(urlParams);
-	ZmZimbraMail.sendRedirect(url);	// will trigger onbeforeunload
+	if (url) {
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/core/ZmZimbraMail.js]: In Classic

- username dropdown contains "Modern Web App" option
- reroutes to Modern Web Client

Comment (auminpatel) [reply WebRoot/js/zimbraMail/core/ZmZimbraMail.js]: shardool has added a comment in the ticket 
https://jira.corp.synacor.com/browse/PREAPPS-4538?focusedCommentId=1330394&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1330394

**Pull request merged**